core/services/ocr2/plugins/ccip/internal/ccipdb/price_service.go

```diff
@@ -134,8 +121,7 @@ func (p *priceService) Close() error {
 }
 
 func (p *priceService) run() {
-	cleanupTicker := time.NewTicker(p.cleanupInterval)
-	updateTicker := time.NewTicker(p.updateInterval)
+	updateTicker := time.NewTicker(utils.WithJitter(p.updateInterval))
```

Comment:
I think this is fixed in ccip but doesn't look like we do a `defer updateTicker.Stop()`